spec/Bex/Behat/ScreenshotExtension/Driver/LocalSpec.php

```diff
@@ -42,6 +44,17 @@ function it_cleans_up_screenshot_directory_when_switch_is_on(
         $this->load($container, $config);
     }
 
+    function it_skips_cleaning_up_screenshot_directory_when_switch_is_on_and_directory_does_not_exist(
+        Filesystem $filesystem, ContainerBuilder $container
+    ) {
+        $config = $this->getDefaultConfig();
+        $config[Local::CONFIG_PARAM_CLEAR_SCREENSHOT_DIRECTORY] = true;
+
+        $filesystem->exists('/path/to/screenshots/')->willReturn(false);
+
```

Comment:
wow you are lightning fast, thank you! I will wait for scrutinizer and travis-ci, then will merge your pr. Thanks again!